docs-new/docs/graphql/core/api-reference/metadata-api/remote-schemas.mdx

```diff
@@ -71,17 +69,15 @@ X-Hasura-Role: admin
 }
 ```
 
-<div id="metadata_add_remote_schema_syntax">
-
-| Key        | Required | Schema             | Description                      |
-|------------|----------|--------------------|----------------------------------|
-| name       | true     | `RemoteSchemaName` | Name of the remote schema        |
-| definition | true     | `RemoteSchemaDef`  | Definition for the remote schema |
-| comment    | false    | Text               | comment                          |
+### Args syntax {#metadata-add-remote-schema-syntax}
 
-</div>
+| Key        | Required | Schema                                                                            | Description                      |
+|------------|----------|-----------------------------------------------------------------------------------|----------------------------------|
+| name       | true     | [RemoteSchemaName](/graphql/core/api-reference/syntax-defs.mdx#remoteschemaname)  | Name of the remote schema        |
+| definition | true     | [RemoteSchemaDef](/graphql/core/api-reference/syntax-defs.mdx#remoteschemadef)    | Definition for the remote schema |
+| comment    | false    | Text                                                                              | comment                          |
 
-## update_remote_schema
+## update_remote_schema {#metadata-update-remote-schema}
 
 `update_remote_schema` is used to update the configuration of a remote
 schema. If the remote schema URL has changed then it will perform a
```

Comment:
please replace the `span` in this text with <em>.

`<span class="title-ref">inconsistent_metadata</span>.` => `<em>inconsistent_metadata</em>`

Reply:
Done